cfme/configure/access_control/__init__.py

```diff
@@ -836,11 +836,7 @@ def create(self, description=None, role=None, tenant="My Company", ldap_credenti
                 not having appropriate permissions OR delete is not allowed
                 for currently selected user
         """
-        if self.appliance.version < "5.8":
-            flash_blocked_msg = ("Description has already been taken")
-        else:
-            flash_blocked_msg = "Description is not unique within region {}".format(
```

Comment:
REQUIRED: This message look different from your message. 
Besides that, you update code inside framework. So, this code can potentially be used by other tests.
So, to recap I have a suspicion that your change will fix one test but break a bunch of other tests. 
Please make sure that other tests won't be broken and include other tests using this method into PRT.
Btw, please look once more at code where this flash message is used.

Reply:
@izapolsk  
 Yes, I agree with you. This method is used in various tests. 
- My observations:
    - If we do not provide description while creating group then it is showing flash message as ```Description can't be blank```.
    - If we provide description which is similar to other existing group then it shows flash message as ```"Description is not unique within region {}".format(self.appliance.server.zone.region.number```   
- I modified this PR in a way where it will not break other test cases.

Please, let me know if you have any other suggestions.
Thank you.